map_view/display-layer-view-state/src/main/resources/display_layer_view_state/style.css

```
@@ -1,27 +1,18 @@
-.panel-region .label {
-  -fx-text-fill: white;
-}
-
-.label {
-  -fx-text-fill: black;
-}
-
-.slider .axis {
-    -fx-tick-label-fill: white;
+.panel-region {
+  -fx-padding: 10;
+  -fx-background-color: rgba(0, 0, 0, 0.5);
+  -fx-max-width: 400;
+  -fx-max-height: 120;
+  -fx-spacing: 6;
+  -fx-alignment: center;
 }
 
-.range-slider .axis {
-    -fx-tick-label-fill: white;
-}
-
-.panel-region .check-box {
-   -fx-text-fill: white;
-}
-
-.panel-region .radio-button {
-   -fx-text-fill: white;
+.panel-label {
+  -fx-text-fill: white;
+  -fx-font-size: 16px;
+  -fx-text-alignment: center;
 }
 
-.color-picker .color-picker-label {
-   -fx-text-fill: black;
+.panel-button {
+  -fx-max-width: 130;
```

Same here, you can make a rule for any button thats inside the style class 'panel-region'. Then there's no need to do `hideLayerButton.getStyleClass().add("panel-button");` etc in the main class.

```suggestion
.panel-region .button {
  -fx-max-width: 130;
```